server/utils/utils.test.ts

```diff
@@ -396,3 +398,23 @@ describe('isValidDate', () => {
     expect(result).toEqual(true)
   })
 })
+
+describe('isPastDate', () => {
+  it('has future date', () => {
+    const tomorrow = moment().add(1, 'days')
+    const day = tomorrow.format('DD')
+    const month = tomorrow.format('MM')
+    const year = tomorrow.format('YYYY')
+    const result = isPastDate(day, month, year)
+    expect(result).toEqual(true)
+  })
+
+  it('has past date', () => {
+    const yesterday = moment().subtract(1, 'days')
+    const day = yesterday.format('DD')
+    const month = yesterday.format('MM')
+    const year = yesterday.format('YYYY')
+    const result = isPastDate(day, month, year)
+    expect(result).toEqual(false)
+  })
+})
```

Comment:
Unsure whether we should be testing for today's date as that would surely be input in error by a user too as a date of birth. However, maybe it's not strictly covered by this validation and we should discuss as a team if we need validation to check the minimum age of a person?

Reply:
Agree with this, not sure if NOMIS would allow it in - but we probably should be gatekeeping any policy requirements at our front door to stop any issues later down the line. Having a quick google looks like you can be criminally charged at 10, but it also looks like YOI's are geared up for 15 - 18 year olds. We'd need to be careful of using arbitrary rules. So one worth discussing with the team as you said 😃